src/common/getters.ts

```diff
@@ -775,6 +442,389 @@ export function getInherit(vc: DatasetWithId): boolean {
   );
 }
 
+const WELL_KNOWN_KEYS = [
```

Comment:
Short and long forms of the well-known keys to filter out. ESS only uses short-form.